docs/README.md

```diff
@@ -16,7 +16,6 @@ Kodi uses CMake as its building system but instructions are highly dependent on
   <a href="README.Linux.md" title="Linux"><img src="resources/linux.svg" height="78"></a>
   <a href="README.macOS.md" title="macOS"><img src="resources/macos.svg" height="78"></a>
   <a href="README.openSUSE.md" title="openSUSE"><img src="resources/opensuse.svg" height="78"></a>
-  <a href="README.RaspberryPi.md" title="Raspberry Pi"><img src="resources/raspberrypi.svg" height="78"></a>
```

Comment:
the repo can be visited through web browsers, so maybe include a stub file to prevent 404s?

Reply:
Should I just include a symlink to the README.Linux.md or what should I do exactly? The link for this is here https://github.com/xbmc/xbmc/tree/master/docs which is just shown as an icon.